client-hc/src/test/java/com/graphhopper/api/Examples.java

```diff
@@ -92,23 +92,29 @@ public void matrix() {
         matrixClient.setKey(apiKey);
 
         GHMRequest ghmRequest = new GHMRequest();
-        ghmRequest.addOutArray("distances");
-        ghmRequest.addOutArray("times");
+        ghmRequest.setOutArrays(Arrays.asList("distances", "times"));
         ghmRequest.putHint("vehicle", "car");
 
-        // init points for a symmetric matrix
-        // List<GHPoint> allPoints = Arrays.asList(new GHPoint(49.6724, 11.3494), new GHPoint(49.6550, 11.4180));
-        // ghmRequest.addAllPoints(allPoints);
+        // Option 1: init points for a symmetric matrix
+        List<GHPoint> allPoints = Arrays.asList(new GHPoint(49.6724, 11.3494), new GHPoint(49.6550, 11.4180));
+        ghmRequest.setPoints(allPoints);
+        MatrixResponse responseSymm = matrixClient.route(ghmRequest);
+        if (responseSymm.hasErrors())
+            throw new RuntimeException(responseSymm.getErrors().toString());
+        // get time from first to second point:
+        // System.out.println(response.getTime(0, 1));
 
+        // Option 2: for an asymmetric matrix do:
+        ghmRequest = new GHMRequest();
+        ghmRequest.setOutArrays(Arrays.asList("distances", "times"));
+        ghmRequest.putHint("vehicle", "car");
```

Comment:
I guess we should no longer recommend using the vehicle hint/parameter, but rather `setProfile("car")`? Here and a few lines above...